packages/cli/src/generateOutputData.js

```diff
@@ -62,8 +59,9 @@ const generateOutputData = (source, params, options) => {
 
     if (outputFormat === 'jscad' || outputFormat === 'js') {
       resolve(source)
-    } else if ((inputFormat === 'jscad' || inputFormat === 'js') &&
-               outputFormat !== 'jscad' && outputFormat !== 'js') {
+    } else {
+//    } else if ((inputFormat === 'jscad' || inputFormat === 'js') &&
```

Comment:
I am scratching my head a bit about this one, was it not needed to handle js/jscad as output ?

Reply:
I thought so as well, but didn’t find any differences.

We need a good test suite for the CLI, but that’s on the TBD list.